src/test/java/com/lpvs/service/GitHubServiceTest.java

```diff
@@ -1079,6 +1123,460 @@ public void testSetErrorCheck__ApiUrlPresentCantAuthorize() {
+            // main test
+            gh_service.commentResults(webhookConfig, List.of(), null);
+
+            // `mocked_instance_gh` verify
+            try {
+                verify(mocked_instance_gh, times(1)).getRepository(webhookConfig.getRepositoryOrganization() + "/" + webhookConfig.getRepositoryName());
+            } catch (IOException e) {
+                LOG.error("TestCommentResults__ScanResultsEmpty.testCommentResults__ScanResultsEmpty() error " + e);
+                fail();
+            }
+            verifyNoMoreInteractions(mocked_instance_gh);
+
+            // `mocked_repo` verify
+            try {
+                verify(mocked_repo, times(1)).getPullRequests(GHIssueState.OPEN);
+                verify(mocked_repo, times(1)).createCommitStatus(commit_sha, GHCommitState.SUCCESS, null,
+                        "No license issue detected", "[Open Source License Validation]");
+            } catch (IOException e) {
+                LOG.error("TestCommentResults__ScanResultsEmpty.testCommentResults__ScanResultsEmpty() error " + e);
+                fail();
+            }
+            verifyNoMoreInteractions(mocked_repo);
+        }
+    }
+
+    @Nested
+    class TestCommentResults__ProhibitedPresentConflictsPresent {
+        final GitHubService gh_service = new GitHubService(null, null, null);
+        GitHub mocked_instance_gh = mock(GitHub.class);
+        GHRepository mocked_repo = mock(GHRepository.class);
+        GHPullRequest mocked_pr_1;
+        GHPullRequest mocked_pr_2;
+        final String url_pr_1 = "https://api.github.com/repos/Samsung/LPVS/pulls/18";
+        final String url_pr_2 = "https://api.github.com/repos/Samsung/LPVS/pulls/19";
+
+        // `webhookConfig`
+        WebhookConfig webhookConfig;
+        final String repo_org = "Samsung";
+        final String repo_name = "LPVS";
+        final String commit_sha = "895337e89ae103ff2d18c9e0d93709f743226afa";
+        final String repository_url = "https://github.com/Samsung/LPVS";
+
+        // `lpvs_file_1`
+        LPVSFile lpvs_file_1;
+        final String file_url_1 = "https://github.com/Samsung/LPVS/tree/main/src/main/java/com/lpvs/service/GitHubService.java";
+        final String file_path_1 = "src/main/java/com/lpvs/service/GitHubService.java";
+        final String snippet_match_1 = "/**\n" +
+                " * Copyright (c) 2022, Samsung Electronics Co., Ltd. All rights reserved.\n" +
+                " *\n" +
+                " * Use of this source code is governed by a MIT license that can be\n" +
+                " * found in the LICENSE file.\n" +
+                " */\n";
+        final String matched_lines_1 = "1-6";
+        final String component_1 = "LPVS::Services";
+
+        // `lpvs_file_1`
+        LPVSLicense lpvs_license_1;
+        final String license_name_1 = "MIT License";
+        final String spdx_id_1 = "MIT";
+        final String access_1 = "PROHIBITED";
```

Comment:
Do we need to explosure the policy (in this code `access`) in public?

Reply:
To cover the code related to the processing of the detected prohibited license and the license conflict, we are simulating such a case in this unit test.
This case is an artificial and has no relation to the real policy.

Reply:
Oleg is right. These are just random values for unit test.
I can change them to other random values, should I do this?

Reply:
Then the presented `access_1` can be anything, right? What about `CASE1` or `CASE2` or something in this case?

Reply:
The value of `access_1` could be one among the list: "PERMITTED", "RESTRICTED", or "PROHIBITED".

Reply:
https://github.com/Samsung/LPVS/blob/443ed9ddeeda83d0da09c6daf878350d33863bf1/src/main/java/com/lpvs/service/GitHubService.java#L176

we need it to get into this `if` branch

Reply:
Is the current implementation dependent on the values in `access` then? Please educate me if this test is in the end for our current implementation in conjunction with our previous discussion in https://github.com/Samsung/LPVS/pull/47. 🙏

Reply:
Conclusion of the LPVS about the usage of prohibited or restricted licenses depends on the file [licenses.json](https://github.com/Samsung/LPVS/blob/main/src/main/resources/licenses.json)
According to the README file, a user has to fill this file by himself before starting to use LPVS. We provide only a template for this file.
Here in unit test we simulate values from this `licenses.json` file using variables `license_name_1`, `spdx_id_1`, `access_1`, etc., which is equivalent that we read them from the file.

Reply:
+1 Thank you for your kind clarification. 😄